ivy/functional/frontends/torch/nn/functional/loss_functions.py

```diff
@@ -334,16 +334,19 @@ def batchmean(x):
 
 
 @to_ivy_arrays_and_back
+@with_unsupported_dtypes(
+    {"2.0.1 and below": ("int8", "int16", "int32", "int64", "bool", "uint8")}, "torch"
+)
```

Comment:
Could be written as 
```
@with_supported_dtypes(
    {"2.0.1 and below": ("float", "complex")}, "torch"
)
```
We've added these keywords for supersetting: "float", "complex", "integer", "unsigned", "numeric".